src/console_cmd.c

```diff
@@ -709,6 +709,548 @@ TbBool cmd_exec(PlayerNumber plyr_idx, char *msg)
                 }
             }
         }
+        else if (strcasecmp(parstr, "room.available") == 0)
+        {
+            long roomid = get_rid(room_desc, pr2str);
+            if (roomid <= 0)
+            {
+                if (strcasecmp(pr2str, "Hatchery" ) == 0)
+                {
+                    roomid = RoK_GARDEN;
+                }
+                else if ( (strcasecmp(pr2str, "Guard" ) == 0) || (strcasecmp(pr2str, "GuardPost" ) == 0) )
+                {
+                    roomid = RoK_GUARDPOST;
+                }
+                else
+                {
+                    roomid = atoi(pr2str);
+                }
+            }
+            unsigned char available = (pr3str == NULL) ? 1 : atoi(pr3str);
+            PlayerNumber id = get_player_number_for_command(pr4str);
+            script_process_value(Cmd_ROOM_AVAILABLE, id, roomid, (TbBool)available, (TbBool)available);
+            update_room_tab_to_config();
+            return true;
+        }
+        else if ( (strcasecmp(parstr, "power.give") == 0) || (strcasecmp(parstr, "spell.give") == 0) )
+        {
+            long power = get_rid(power_desc, pr2str);
+            if (power <= 0)
+            {
+                if ( (strcasecmp(pr2str, "Imp" ) == 0) || (strcasecmp(pr2str, "CreateImp" ) == 0) )
```

Comment:
Is this whole bit still needed now you've fixed the _? I think it would be best to just take the correct power name. So in this case POWER_IMP.

Reply:
Probably not.

Reply:
So, could you clean it up?